storefront/modules/cart/services/CartService.ts

```diff
@@ -1,54 +1,104 @@
+import { CartItemGetVm, CartItemGetDetailsVm } from '../models/CartItemGetVm';
 import { ProductThumbnail } from '../../catalog/models/ProductThumbnail';
-import { AddToCartModel } from '../models/AddToCartModel';
-import { Cart } from '../models/Cart';
-import { UpdateCartModel } from '../models/UpdateCartModel';
 import apiClientService from '@/common/services/ApiClientService';
+import { CartItemPostVm } from '../models/CartItemPostVm';
+import { YasError } from '@/common/services/errors/YasError';
+import { getProductsByIds } from '@/modules/catalog/services/ProductService';
+import { CartItemPutVm } from '../models/CartItemPutVm';
+import { CartItemDeleteVm } from '../models/CartItemDeleteVm';
 
-const baseUrl = '/api/cart';
+const CART_BASE_URL = `/api/cart/storefront/cart/items`;
```

Comment:
This should not be named `CART_BASE_URL`, Let's use a different name.

Reply:
The route `/api/cart/storefront/cart/items` is specifically for the cart service, and I don’t have a new name in mind right now. Do you have any suggestions? Thank you!

Reply:
Let's keep as is.